Modules/Filtering/Thresholding/include/itkHuangThresholdCalculator.hxx

```diff
@@ -22,6 +22,7 @@
 #include "itkMath.h"
 #include "itkProgressReporter.h"
 #include "itkMath.h"
+#include <cmath> // For round.
```

Comment:
"itkMath.h" is included twice, BTW

Reply:
@issakomi Interesting... It appears quite common to `#include "itkMath.h"` twice! Just did a search for regular expression `#include "itkMath\.h"\r\n.*#include "itkMath\.h"`... 18 hits!  😸  Feel free to make that a new PR, if you like!

Reply:
Interesting. And the winner is [itkBSplineScatteredDataPointSetToImageFilter.hxx](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Filtering/ImageGrid/include/itkBSplineScatteredDataPointSetToImageFilter.hxx) with 3 `#include "itkMath.h"
`

Reply:
_itkMath.h_ already has `#include <cmath>`, BTW.

Reply:
S. #4436

Reply:
> _itkMath.h_ already has `#include <cmath>`, BTW.

OK, very well observed, but in general I would rather not depend on _indirectly_ included headers. Following the "Include What You Use" philosophy. (An indirectly_ included header might just be an internal implementation detail of the directly included header.)